website/docs/scaffolds/nft-scaffold.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: The NFT Scaffold Project
+---
+
+| Scaffold Details                                                          |         
+|--------------------|------------------------------------------------------|
+| Complexity         | Advanced                                             |
+| Automated Tests    | No                                                   |
+| Installed Plugins  | LIGO, Taquito, Flextesa, Contract Types, IPFS-Pinata |
+| Frontend Dapp      | Yes                                                  |
+| Wallet Integration | Yes                                                  |
+
+:::warning
+The NFT-Scaffold project is currently in alpha. It is not ready for production use and will be undergoing frequent updates as we work to implement Taqueria State and other features that have not yet been released
+:::
+
+## Project Setup 
+
+1. Scaffold the project by running:
+
+```bash
+taq scaffold https://github.com/ecadlabs/taqueria-scaffold-quickstart quickstart-project 
```

Comment:
The should be `https://github.com/ecadlabs/taqueria-scaffold-nft`.

The output directory should be `nft-quickstart`.